fluent-react/src/localized.js

```diff
@@ -82,18 +82,23 @@ export default class Localized extends Component {
   render() {
     const { l10n } = this.context;
     const { id, attrs, children } = this.props;
-    const elem = Children.only(children);
+
+    // Validate that children isn't an array
+    if (Array.isArray(children)) {
+      throw new Error("<Localized/> expected to receive a single " +
+        "React node child");
+    }
 
     if (!l10n) {
       // Use the wrapped component as fallback.
-      return elem;
+      return children;
```

Comment:
I'd prefer to keep the previous version with `elem`, please. It make it clearer that there should only be one child element. Same comment applies to other places in this PR.